### ISSUE TITLE
DAR101 false positive

### ISSUE THREAD

**Issue by Trim21** [terrencepreilly/darglint#65]
```python
    @abstract.abstractmethod
    def __init__(self, config: dict):
        """
        :param config: config dict user defined in config file.
        """
```
```python
    @abstract.abstractmethod
    def __init__(self, config: dict):
        """

        :param config: config dict user defined in config file.
        """
```
DAR101 Missing parameter(s) in Docstring: - config

worked with 
```diff
    @abstract.abstractmethod
    def __init__(self, config: dict):
-        """
+        """anything

        :param config: config dict user defined in config file.
        """
```

**Comment by terrencepreilly** [comment]
It's mistaking the line for the short description.  To make the parser more efficient, I split the docstrings up and handle sections separately.  Currently, it assumes the first item in the docstring is a short description.

Since all docstrings should have a short description, I don't imagine it's a very big problem.  But it's still something that should be fixed.  It should be pretty easy to fix -- just add a check for a colon in the first section, and if that's present then try all of the other parsers before the short description.

**Comment by terrencepreilly** [comment]
Ah, it looks like I handled this in 6b28770b1299a8307adbc4cf7348f7271365f51f, but forgot to link it to this bug.  Fix should be present in v1.2.2.
